engine/src/module-engine.cpp

```diff
@@ -316,11 +319,16 @@ extern "C" MC_DLLEXPORT_DEF void MCEngineExecGetPropertyOfScriptObject(MCStringR
     MCExecContext ctxt(MCdefaultstackptr, nil, nil);
 
     // AL-2015-07-24: [[ Bug 15630 ]] Syntax binding dictates value returned as out parameter rather than directly
-	r_value = MCEngineGetPropertyOfObject(ctxt, p_property, t_object, t_part_id);
+	r_value = MCEngineGetPropertyOfObject(ctxt, p_set, p_property, t_object, t_part_id);
+}
+
+extern "C" MC_DLLEXPORT_DEF void MCEngineExecGetPropertyOfScriptObject(MCStringRef p_property, MCScriptObjectRef p_object, MCValueRef& r_value)
+{
+	MCEngineExecGetPropertyOfSetOfScriptObject(p_property, kMCEmptyString, p_object, r_value);
 }
 
 // IM-2015-02-23: [[ WidgetPopup ]] Factored-out function for setting the named property of an object to a value.
-void MCEngineSetPropertyOfObject(MCExecContext &ctxt, MCStringRef p_property, MCObject *p_object, uint32_t p_part_id, MCValueRef p_value)
+void MCEngineSetPropertyOfObject(MCExecContext &ctxt, MCStringRef p_set, MCStringRef p_property, MCObject *p_object, uint32_t p_part_id, MCValueRef p_value)
```

Comment:
I was going to ask if it was possible to restructure this so that we don't need to call `MCValueRelease` in each failure branch, but actually I think it's quite awkward either way :-(